src/middleware/initiate.js

```diff
@@ -4,12 +4,37 @@ const uuid = require('uuid')
 
 const logger = require('../logger')
 
-exports.initiateContextMiddleware = metaData => async (ctx, next) => {
+const {constructOpenhimResponse} = require('../openhim')
+const {endpointCache} = require('../db/services/endpoints/cache')
+
+const getEndpoint = urlPath => {
```

Comment:
minor: Maybe update this function name to state the variable we require for the lookup. `getEndpointByPath`